rust/gitxetcore/src/constants.rs

```diff
@@ -17,6 +17,8 @@ pub const GIT_NOTES_REPO_SALT_REF_NAME: &str = "refs/notes/xet/reposalt";
 
 pub const GIT_LAZY_CHECKOUT_CONFIG: &str = "xet/lazyconfig";
 
+pub const GIT_REPO_SPECIFIC_CONFIG: &str = ".xet/config.toml";
```

Comment:
Let's add a comment here says that this file is checked-in (in the working directory).